src/main/scala/com.snowplowanalytics.sauna/responders/pagerduty/CreateEventResponder.scala

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2016-2017 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.sauna
+package responders
+package pagerduty
+
+// scala
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.io.Source
+import scala.util.{Failure, Success}
+
+// akka
+import akka.actor.{ActorRef, Props}
+
+// play
+import play.api.libs.json.Json
+
+// sauna
+import apis.PagerDuty
+import apis.PagerDuty.PagerDutyEvent
+import loggers.Logger.Notification
```

Comment:
I'm getting `[error] bad option: '-Ywarn-unused-import'` - that flag might have been replaced by something else.